python.d/postgres.chart.py

```diff
@@ -208,17 +291,52 @@
             ['table_size', 'size', 'absolute', 1, 1024 * 1024]
         ]},
     'wal': {
-        'options': [None, 'Write-Ahead Logging Statistics', 'files/s', 'write ahead log', 'postgres.wal', 'line'],
+        'options': [None, 'Write-Ahead Logs', 'files/s', 'wal', 'postgres.wal', 'line'],
+        'lines': [
+            ['written_wal', 'written', 'absolute'],
+            ['recycled_wal', 'recycled', 'absolute'],
+            ['total_wal', 'total', 'absolute']
+        ]},
+    'archive_wal': {
+        'options': [None, 'Archive Write-Ahead Logs', 'files/s', 'archive wal', 'postgres.archive_wal', 'line'],
         'lines': [
             ['file_count', 'total', 'incremental'],
             ['ready_count', 'ready', 'incremental'],
             ['done_count', 'done', 'incremental']
         ]},
-    'background_writer': {
-        'options': [None, 'Checkpoints', 'writes/s', 'background writer', 'postgres.background_writer', 'line'],
+    'checkpointer': {
+        'options': [None, 'Checkpoints', 'writes/s', 'checkpointer', 'postgres.checkpointer', 'line'],
+        'lines': [
+            ['checkpoint_scheduled', 'scheduled', 'incremental'],
+            ['checkpoint_requested', 'requested', 'incremental']
+        ]},
+    'bgwriter': {
+        'options': [None, 'Buffers', 'kilobytes/s', 'bgwriter', 'postgres.bgwriter', 'line'],
+        'lines': [
+            ['buffers_checkpoint', 'checkpoint', 'incremental', 1, 1024],
+            ['buffers_clean', 'clean', 'incremental', 1, 1024],
+            ['maxwritten_clean', 'maxwritten', 'incremental'],
+            ['buffers_backend', 'backend', 'incremental', 1, 1024],
+            ['buffers_alloc', 'alloc', 'incremental', 1, 1024],
+            ['buffers_backend_fsync', 'backend_fsync', 'incremental']
+        ]},
+    'autovacuum': {
+        'options': [None, 'Autovacuum workers', 'workers', 'autovacuum', 'postgres.autovacuum', 'line'],
         'lines': [
-            ['writer_scheduled', 'scheduled', 'incremental'],
-            ['writer_requested', 'requested', 'incremental']
+            ['autovacuum_max_worker', 'max workers', 'absolute'],
+            ['analyze', 'analyze', 'absolute'],
+            ['vacuum', 'vacuum', 'absolute'],
+            ['vacuum_analyze', 'vacuum_analyze', 'absolute'],
+            ['vacuum_freeze', 'vacuum_freeze', 'absolute'],
+            ['brin_summarize', 'brin_summarize', 'absolute']
+        ]},
+    'secondary_delta': {
+        'options': [None, 'Secondary delta', 'kilobytes/s', 'replication delta', 'postgres.secondary_delta', 'line'],
+        'lines': [
+            ['sent_delta', 'sent_delta', 'absolute', 1, 1024],
+            ['write_delta', 'write_delta', 'absolute', 1, 1024],
+            ['flush_delta', 'flush_delta', 'absolute', 1, 1024],
+            ['replay_delta', 'replay_delta', 'absolute', 1, 1024]
```

Comment:
if `delta` is needed in dimension i suggest to use space instead of underscore. It looks better on the dashboard.
```pyhon
 ['replay_delta', 'replay delta', 'absolute', 1, 1024]
```
Same true for all new dimensions with underscore.

Another problem: netdata applies interpolation to find the exact value at second-boundaries only for `incremental `values.  `/s` is misleading for `absolute` (there is no guarantee that the module runs once a second).